app/views/includes/organisation-overview.html

```diff
@@ -74,6 +75,11 @@
           <td headers="current cap">{{ member.capacityPercentage | round }}%</td>
         {% endif %}
         <td headers="points cap">{{ member.availablePoints }}</td>
+        {% if member.capacityPercentage > 110 %}
+          <td class="capacity-red" headers="remaining cap">{{ member.availablePoints - member.totalPoints }}</td>
```

Comment:
`capacity-red` has introduced pa11y errors. 

` • Error: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4:1. Recommendation: change text colour to #e00.`

To run pa11y tests, start the app and run `yarn accessibility-test` in a separate terminal window